src/index.js

```diff
@@ -1 +1,83 @@
 import './style.css';
+import Shows from './shows.js';
+import { showsContainer, searchForm, searchFormInput } from './dynamic.js';
+import likes from './images/Empty_Like.svg';
+
+const setNewShow = (programList) => {
+  let tvShows = [];
+  programList.forEach((program) => {
+    const { id, image } = program.show;
+    const title = program.show.name;
+    const like = false;
+    const newShow = new Shows(id, image, title, like);
+    if (image) {
+      tvShows = [...tvShows, newShow];
+    }
+  });
+  return tvShows;
+};
+
+const getTVShows = async (search) => {
+  if (search === undefined) {
+    search = 'action';
+  }
+  const resultPrograms = await fetch(
+    `https://api.tvmaze.com/search/shows?q=${search}`,
+  );
+  const programList = await resultPrograms.json();
+  const newProgramList = setNewShow(programList);
+  if (newProgramList.length <= 0) {
+    searchFormInput.placeholder = 'Please try with another TV Show or Genre';
+    searchFormInput.classList.add('error_input_show');
+  }
+  return newProgramList;
+};
```

Comment:
[Optional] kindly add this function in a different js file and import into this file (index.js)